pages/_app.js

```diff
@@ -238,7 +202,7 @@ export default function App({ Component, pageProps }) {
 
       <ThemeProvider theme={theme}>
         {router.pathname.split("/").reverse()[0] === "login" ? null : (
-          <Header />
+          <Header props = {loggedOut ? { loggedOut } : {}} />
```

Comment:
By setting it up this way, you will be passing different props to `Header` on *every* render of `App` even if the value of `loggedOut` doesn't change. This is because you're creating a new object each time. So, why not just used `loggedOut` instead and update `Header`, accordingly to avoid this?
```suggestion
          <Header loggedOut={loggedOut} />
```